app/controllers/content_items_controller.rb

```diff
@@ -26,8 +26,6 @@ def present(content_item)
     presenter_name = content_item['schema_name'].classify + 'Presenter'
     presenter_class = Object.const_get(presenter_name)
     presenter_class.new(content_item)
-  rescue NameError
```

Comment:
This sort of change should be in a separate commit with a clear reason.
I don't think it belongs in this PR.